client/src/pages/LoginPage.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { useAuth0 } from "@auth0/auth0-react";
+import { Button, Flex, Image, VStack, useColorModeValue } from "@chakra-ui/react";
+import Logo from "../assets/breteau_logo.png";
+
+const LoginPage = () => {
+	const { loginWithPopup } = useAuth0();
+	const containerBg = useColorModeValue("teal.200", "teal.300");
+	return (
+		<VStack alignItems="center" justifyContent="center" minH="80vh">
+			<Flex flexDirection="column" alignItems="center" bgColor={containerBg} px="2rem" py="1rem" borderRadius="2xl" shadow="2xl">
+				<Image src={Logo} mb="10" width="17.5rem" />
+				<Button onClick={() => loginWithPopup()} size="lg" colorScheme="blue">
```

Comment:
out of curiosity, why `loginWithPopup` and not `loginWithRedirect`?

Reply:
no specific reason, thought it looked nicer with a prompt rather than a redirect. 

I think it's giving me issues on heroku so I have changed it to redirect and will see if it fixes the issue